psi4/src/psi4/libsapt_solver/fdds_disp.h

```diff
@@ -60,34 +60,64 @@ class FDDS_Dispersion {
     std::map<std::string, SharedMatrix> matrix_cache_;
     std::map<std::string, SharedVector> vector_cache_;
 
+    // Is hybrid functional? 
+    bool is_hybrid_;
+
+    // QR factorization result
+    SharedMatrix R_A_, R_B_;
+
    public:
     /**
      * Constructs the FDDS_Dispersion object.
      * @param primary   The primary basis
      * @param auxiliary The auxiliary basis
      * @param cache     A data cache containing "Cocc_A", "Cvir_A", "eps_occ_A", "eps_vir_A", "Cocc_B",
      * "Cvir_B", "eps_occ_B", "eps_vir_B" quantities
+     * @param is_hybrid Flag of hybrid functional
      */
     FDDS_Dispersion(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> auxiliary,
-                    std::map<std::string, SharedMatrix> matrix_cache, std::map<std::string, SharedVector> vector_cache);
+                    std::map<std::string, SharedMatrix> matrix_cache, std::map<std::string, SharedVector> vector_cache,
+                    bool is_hybrid);
 
     ~FDDS_Dispersion();
 
     /**
      * Projects the densities from primary AO space to auxiliary AO space
      * @param  dens     Vector of densities to transform
-     * @return axu_dens Vector of transformed densities
+     * @return aux_dens Vector of transformed densities
      */
     std::vector<SharedMatrix> project_densities(std::vector<SharedMatrix> dens);
 
     /**
-     * Forms the uncoupled amplitude Qia,Eia,Pia->PQ
+<<<<<<< HEAD
+<<<<<<< HEAD
+     * Forms the uncoupled amplitude Qia,Eia,Pia->PQ 
+=======
+     * Forms the uncoupled amplitude Qia,Eia,Pia->PQ and projector Qia,Pia->PQ
+>>>>>>> update hybrid-DFT FDDS dispersion
+=======
+     * Forms the uncoupled amplitude Qia,Eia,Pia->PQ 
+>>>>>>> add DFHelper::clear_transformation() and reform FDDS dispersion integrals
```

Comment:
this probably runs fine b/c it's in a comment, but this is a merge conflict to resolve.

Reply:
Thanks for finding this out! I somehow didn't see any warning about this... Will fix this soon.